Makefile

```diff
@@ -161,6 +161,7 @@ test_unit: ## run all unit tests
 .PHONY: functest
 functest: ## run functest FUNCTEST_COUNT times
 	CGO_ENABLED=1 $(GOTEST) -test.v $(TEST_ARGS) -tags "functest bloattest" ./application/functest -count=$(FUNCTEST_COUNT)
+	sleep 10 # workaround for a "bind: address already in use" problem
```

Comment:
I want to say, that the highest standards continue to live

Reply:
I feel quite sad when you are no longer part of Insolar. But, I always thank you for all that you have devoted to Insolar. Wishing you much success in the way to come!